### DISCUSSION TITLE
【玩家招募】有没有一起玩我的世界服务器的

### DISCUSSION THREAD

**Discussion by H-luoyan** [share121/inter-knot#2691] [其他]
1.21版本服务器，连接地址111.161.122.206:49170，QQ群916641081，欢迎大家
![Uploading 6abe34c165a5f67d20c04bc663d6bd1.png…]()

**Comment by jsishsjsj** [comment]
直接发ip有点危险啊绳酱<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >，建议你把ip去了，免得被D

**Comment by scrooge-0** [comment]
同意，Java非正版可以来吗？ @H-罗燕

**Reply by H-luoyan** [reply]
非正版可以的
